src/main/java/weavers/siltarae/mistake/dto/request/MistakeCreateRequest.java

```diff
@@ -0,0 +1,21 @@
+package weavers.siltarae.mistake.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class MistakeCreateRequest {
+    private List<Long> tagIds;
+    @NotNull(message = "실수 내용은 필수 입니다.")
+    @Size(max = 140, message = "실수 내용은 140자를 넘을 수 없습니다.")
+    private String content;
+}
```

Comment:
한 줄 띄워주시면 가독성이 더 좋아질 것 같은데 매니님의 생각이 궁금합니다~!

```suggestion
public class MistakeCreateRequest {
    private List<Long> tagIds;
    
    @NotNull(message = "실수 내용은 필수 입니다.")
    @Size(max = 140, message = "실수 내용은 140자를 넘을 수 없습니다.")
    private String content;
}
```